src/Altinn.Apps/AppFrontend/react/altinn-app-frontend/src/features/confirm/containers/Confirm.tsx

```diff
@@ -106,12 +103,6 @@ const Confirm = (props: IConfirmProps) => {
     InstanceDataActions.getInstanceData(routeParams.partyId, routeParams.instanceGuid);
   }, []);
 
-  React.useEffect(() => {
```

Comment:
where do we get the language from when this effect is removed?

Reply:
The language files are fetched by the user profile settings. This is stored in redux. No need to find this setting here.